sdk/storage/azdatalake/file/client_test.go

```diff
@@ -4237,18 +4231,8 @@ func (s *RecordedTestSuite) TestFileUploadDownloadSmallFile() {
 	_require.NoError(err)
 	_require.Equal(cnt, fileSize)
 
-	err = destFile.Close()
```

Comment:
Add a defer method for closing the file handle in `TestFileUploadDownloadSmallFile` and `TestFileUploadDownloadSmallFileWithRange`.

Reply:
missed this. Thanks! 
Added.